extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventCommonFieldsMixin.schema.json

```diff
@@ -157,6 +157,51 @@
               "title": "batchUnitaryBranchID",
               "description": "If the instance has been triggered from a batch instance, unitary branch ID.",
               "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/journeyNodeName": {
+              "title": "journeyNodeName",
```

Comment:
Titles need to be proper sentences with each starting letter upper case. Same comment applies to the other titles as well.